docs/docs/setting-up-ci-cd.mdx

```diff
@@ -38,20 +38,22 @@ you can make a test run only if the pull request has a certain label (e.g. “NL
 
 ### Validating Data and Stories
 
-Data validation verifies that there are no mistakes or major inconsistencies in your domain, NLU 
-data, or conversation data. To validate your data, have your CI run this command:
+Data validation verifies that no mistakes or major inconsistencies appear in your domain, NLU 
+data, or story data. To validate your data, have your CI run this command:
```

Comment:
What if you have a rule that contains an action/intent not in your domain? Does data validation catch that?

Reply:
Yes, it should catch that. Just not consistency conflicts with stories. I'll add a test for this, too.

Reply:
@JEM-Mosig just so I understand it 100%: `rasa data validate` validates stories, intents, rules, etc., but not the stories-rules consistency. And `rasa data validate stories` checks only stories and doesn't look at rules at all. Correct?

Reply:
No, `rasa data validate` does everything, including story consistency. `rasa data validate stories` only does story consistency. We don't have a tool that does rule-story consistency, except the check during rule policy training.

Reply:
Yeah, that's what I meant. Cool :-)